c2corg_api/security/discourse_client.py

```diff
@@ -53,10 +53,16 @@ def discourse_get_username_by_userid(client, userid):
 def discourse_sync_sso(user, settings):
     key = str(settings.get('discourse.sso_secret'))  # must not be unicode
     client = get_discourse_client(settings)
+    if user.forum_username is None:
```

Comment:
When you register a new account, would you have to provide at least two different user names?